public/cli/lib/commands/convert.js

```diff
@@ -410,69 +427,130 @@ function removeOutputFile(done) {
+                    if (Options.onProgress) {
+                    opts.onProgress = onProgress;
+                    }
+                    opts.Log = Log;
+                    Log("Options: ", Options);
+                    Log("opt: ", opts);
+                    onProgress("Rendering video frames...", 0);
+                    return Frames.run(opts);
+                })
+                .then((pathFrames) => {
+                    Log(`>> path to generated frames folder: ${pathFrames}`);
+        
+                    let output;
+                    let outputFilename = path.parse(Options.output).name;
+            
+                    if (Options.combineOutput) {
+                        output = path.join(tempy.directory(), `${outputFilename}-${index+1}.mp4`);
+                        outputFiles.push(output);
+                    }
+                    else {
+                        // when --combineOutput is set to false, then generate multiple files.
+                        // Mark1.mp4, Mark2.mp4, Mark3.mp4, Mark4.mp4
+                        if (Options.pathFolders.length > 1) {
+                            let outputDir = path.parse(Options.output).dir;
+                            output = `${outputDir}/${outputFilename}${index+1}.mp4`;
+                        }
+                        // --combineOutput == false and pathFolders has only 1 item
+                        else {
+                            output = Options.output;
+                        }
+                    }
+
+                    onProgress("Combining audio and frames into video...", 100);
+                    return FFMPEG.run({
+                        images: pathFrames,
+                        audio: path.dirname(pathToInfo),
+                        skipAudioFiles,
+                        output,
+                        Log,
+                        framerateOut: Options.fps || 15,
+                        ffmpegPath: Options.ffmpegPath || null
+                    });
+                })
+            );
+    });
+
+    if (Options.combineOutput) {
+        let outputListFile;
+
+        // Prepare a video list file to combine
+        tasks.push(() => new Promise((next, fail) => {
+            outputListFile = path.join(tempy.directory(), "outputList.txt");
+            Log(`>> path to the video list file: ${outputListFile}`);
+
+            fs.writeFile(
+                outputListFile,
+                outputFiles.map((f) => `file '${f}'`).join("\n"),
+                (err) => {
+                    return err ? fail(err): next();
+                });
+        }));
+
+        // Combine videos
+        tasks.push(() => new Promise((next, fail) => {
+            let ffmpegExe = Options.ffmpegPath || "ffmpeg";
+            Log(`>> Combining videos`);
+
+            onProgress("Combining videos...", 100);
+            shell.exec(
+                `"${ffmpegExe}" -f concat -safe 0 -i "${outputListFile}" -c copy "${Options.output}"`,
```

Comment:
This seems like a good way to combine videos using FFMPEG.  Here is a document which talks about different approaches (for research purposes): https://trac.ffmpeg.org/wiki/Concatenate